spec/system/task_spec.rb

```diff
@@ -1,24 +1,25 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { create(:project) }
+  let(:task) { create(:task, project_id: project.id) }
```

Comment:
Factoryに`association`を設定すると`project_id`の指定は省略できます！

コード量を減らして見通しが良くなるので試してみてください。
```suggestion
  let(:task) { create(:task) }
```

Reply:
fc7f9ca でアソシエーションを設定しました。